api/src/main/java/org/openmrs/module/ugandaemr/metadata/core/EncounterTypes.java

```diff
@@ -308,4 +308,20 @@ public String uuid() {
             return "afcdfcd8-defe-11eb-ba80-0242ac130004";
         }
     };
+
+    public static EncounterTypeDescriptor ANC_CARD_ENCOUNTER_TYPE = new EncounterTypeDescriptor() {
```

Comment:
@Mwanje You dont need this descriptor as the encounter is already existing

Reply:
Done